src/libs/firebase/firestore/featureFlags/getFeatureFlag.ts

```diff
@@ -5,13 +5,14 @@ import { captureMonitoringError } from 'libs/monitoring'
 
 export const getFeatureFlag = (
   featureFlag: RemoteStoreFeatureFlags
-): Promise<void | { minimalBuildNumber: number } | null> =>
+): Promise<void | { minimalBuildNumber?: number; maximalBuildNumber?: number } | null> =>
   firestoreRemoteStore
     .collection(RemoteStoreCollections.FEATURE_FLAGS)
     .doc(env.ENV)
     .get()
     .then((docSnapshot) => {
-      const firebaseFeatureFlag = docSnapshot.get<{ minimalBuildNumber: number }>(featureFlag)
+      const firebaseFeatureFlag =
+        docSnapshot.get<{ minimalBuildNumber: number; maximalBuildNumber?: number }>(featureFlag)
```

Comment:
c'est étonnant que TypeScript ne rale pas

```suggestion
        docSnapshot.get<{ minimalBuildNumber?: number; maximalBuildNumber?: number }>(featureFlag)
```

on n'a pas eu le temps de finir de l'écrire, hier

Reply:
J'ai refacto pour réutiliser ce type dans le retour de la fonction et le `get`